src/map.c

```diff
@@ -1,93 +1,83 @@
 #include "common.h"
 #include "sys/map.h"
 
-// .data start: 80092a60 ?
-// .bss start: 800b49f0 ?
```

Comment:
Could you add these comments back? I think the merge removed them. Handy to have until we get those mapped in splat, even if they're kinda questions

Reply:
Oops I must've "deleted" those when solving merge conflicts, restored those